v2.1/import-data.md

```diff
@@ -77,10 +77,94 @@ For reference, CockroachDB uses these defaults:
+### Step 1. Dump the MySQL database
+
+Issue a command like the following (modified as needed for your environment):
+
+{% include copy-clipboard.html %}
+~~~ sh
+$ mysqldump -uroot --tab=/tmp employees --fields-enclosed-by='"' --fields-escaped-by='\'
+~~~
+
+Always specify an escape character when running `mysqldump --tab`. If your data happens to include any of the special characters, the dumped data cannot be decoded correctly if they are not escaped. We also recommend specifying an enclose character.
+
+Your MySQL user will need permission to execute [`SELECT INTO OUTFILE`](https://dev.mysql.com/doc/refman/8.0/en/select.html), as required by [`mysqldump --tab`](https://dev.mysql.com/doc/refman/8.0/en/mysqldump-delimited-text.html).
+
+If you get the following error, you will need to change the value of MySQL's [`secure_file_priv` system variable](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_secure_file_priv):
+
+~~~
+mysqldump: Got error: 1290: The MySQL server is running with the --secure-file-priv option so it cannot execute this statement when executing 'SELECT INTO OUTFILE'
+~~~
+
+### Step 2. `IMPORT` the dump files
+
+To import the MySQL dump file for a table, issue an `IMPORT` statement like the one shown below. You must use the same settings for field enclosure and escape characters that you provided to `mysqldump --tab` in the previous step.
+
+You will need to look at the dump's `foo.sql` file that corresponds to the `foo.txt` data file and translate the MySQL `CREATE TABLE` statement there into something CockroachDB understands.
+
+This example uses S3. For a complete list of the types of cloud storage `IMPORT` can pull from, see [Import File URLs](import.html#import-file-urls).
+
+{% include copy-clipboard.html %}
+~~~ sql
+IMPORT TABLE employees (
+  emp_no INT PRIMARY KEY,
+  birth_date DATE NOT NULL,
+  first_name STRING NOT NULL,
+  last_name STRING NOT NULL,
+  gender STRING NOT NULL,
+  hire_date DATE NOT NULL
+)
+MYSQLOUTFILE DATA ('s3://nate-external-storage/employees.txt?AWS_ACCESS_KEY_ID=ACCESSKEY&AWS_SECRET_ACCESS_KEY=SECRET')
+~~~
+
+Success will look something like the following:
+
+~~~
++--------------------+-----------+--------------------+------+---------------+----------------+-------+
+|       job_id       |  status   | fraction_completed | rows | index_entries | system_records | bytes |
++--------------------+-----------+--------------------+------+---------------+----------------+-------+
+| 352938237301293057 | succeeded |                  1 |    0 |             0 |              0 |     0 |
++--------------------+-----------+--------------------+------+---------------+----------------+-------+
+(1 row)
+
+Time: 2.129808512s
+~~~
+
+To load the data from local files, you have 2 options:
+
+1. Start up a [local HTTP server](create-a-file-server.html#using-caddy-as-a-file-server) to serve the files, and point `MYSQLOUTFILE DATA` at the server's URL. This is the recommended option.
```

Comment:
I'd make this a bulleted list to avoid people misunderstanding this as steps.

Reply:
nice catch, thanks!